src/lib/API/Event/SchemaBuilderEvents.php

```diff
@@ -8,9 +8,7 @@
 
 namespace EzSystems\DoctrineSchema\API\Event;
 
-use Symfony\Component\EventDispatcher\Event;
-
-class SchemaBuilderEvents extends Event
+class SchemaBuilderEvents
```

Comment:
@adamwojs 
Question: I vaguely remember that convention had changed between Symfony versions and they no longer use dedicated classes with event constants. If that is the case already for Sf 4.x, then I'd suggest we align it here as well.